Slider/Assets/Scripts/UI/Artifact/ArtifactTileButton.cs

```diff
@@ -230,6 +235,28 @@ protected virtual void SetAnchoredPos(int x, int y)
         }
     }
 
+    public void OnSelect()
+    {
+        //Debug.Log(Player.GetInstance().GetComponent<PlayerInput>().currentControlScheme);
+
+        Debug.Log(gameObject.name + " button selected");
+        //SetSelected(true);
+        //SetSpriteToHover();
+        //SetHighlighted(true);
+        if (Player.GetInstance().GetCurrentControlScheme() == "Controller")
+        {
+            SetControllerHoverHighlighted(true);
```

Comment:
edge case where if someone selects a button with controller -> goees to keyboard & mouse -> the highlight will persist?